docs/core/tools/csproj.md

```diff
@@ -194,9 +194,13 @@ A human-friendly title of the package, typically used in UI displays as on nuget
 ### Authors
 A semicolon-separated list of packages authors, matching the profile names on nuget.org. These are displayed in the NuGet Gallery on nuget.org and are used to cross-reference packages by the same authors.
 
-### Description
+### PackageDescription
+
 A long description of the package for UI display.
 
+### Description
+A long description for an assembly. This property is used if `PackageDescription` is not specified.
```

Comment:
We're close 😄 

As it stands right now, it just sounds like the `Description` property is ignored if the `PackageDescription` property is set. I think we need it made clear that we're saying this property will **also** be used for the generated package if the `PackageDescription` is not specified.

Also, I agree with you about all of the descriptions.

Reply:
OK, how does this sound?